experimental/FTheoryTools/src/G4Fluxes/attributes.jl

```diff
@@ -82,12 +82,49 @@ julia> g4_class = (-3) // kbar3(qsm_model) * (5 * e1 * e4 + pb_Kbar * (-3 * e1 -
 julia> g4 = g4_flux(qsm_model, g4_class, check = false)
 G4-flux candidate
   - Elementary quantization checks: not executed
-  - Tadpole cancellation check: not executed
   - Verticality checks: not executed
   - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole cancellation check: not executed
 
 julia> d3_tadpole_constraint(g4, check = false)
 12
+
+julia> qsm_model = literature_model(arxiv_id = "1903.00009", model_parameters = Dict("k" => 2021))
+Hypersurface model over a concrete base
+
+julia> gfs = special_flux_family(qsm_model, check = false)
+A family of G4 fluxes:
+  - Elementary quantization checks: satisfied
+  - Verticality checks: failed
+  - Non-abelian gauge group: broken
+  - Tadpole constraint: not analyzed
+
+julia> g4_2 = random_flux_instance(gfs, check = false)
+G4-flux candidate
+  - Elementary quantization checks: satisfied
+  - Verticality checks: not executed
+  - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole cancellation check: not executed
+
+julia> dv2 = d3_tadpole_constraint(g4_2, check = false);
+
+julia> int_comb = integral_coefficients(g4_2);
+
+julia> rat_comb = rational_coefficients(g4_2);
+
+julia> g4_3 = flux_instance(gfs, int_comb, rat_comb, check = false)
+G4-flux candidate
+  - Elementary quantization checks: satisfied
+  - Verticality checks: not executed
+  - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole cancellation check: not executed
+
+julia> d3_tadpole_constraint(gfs, check = false);
+
+julia> dv3 = d3_tadpole_constraint(g4_3, check = false);
+
+julia> dv2 == dv3
+true
 ```
 """
 function d3_tadpole_constraint(gf::G4Flux; check::Bool = true)
```

Comment:
```suggestion
@attr QQFieldElem function d3_tadpole_constraint(gf::G4Flux; check::Bool = true)
```
this function could make use of `@attr` as well